starknet-replay/src/error/histogram.rs

```diff
@@ -0,0 +1,30 @@
+//! This file contains the enum `Error` for all the errors returned by the
+//! module `histogram`.
+
+use std::num::TryFromIntError;
+
+use plotters::drawing::DrawingAreaErrorKind;
+use thiserror::Error;
+
+#[derive(Debug, Error)]
+pub enum Error {
+    /// `Drawing` variant is for errors reported by the crate `plotters`.
+    #[error(transparent)]
+    Drawing(#[from] DrawingAreaErrorKind<std::io::Error>),
+
+    /// `MathConversion` variant is for errors triggered when calling
+    /// `try_from`.
+    #[error(transparent)]
+    MathConversion(#[from] TryFromIntError),
+
+    /// `MathOverflow` variant is for overflows during math calculations.
+    #[error("Overflow during computation of {0}")]
+    MathOverflow(String),
```

Comment:
Neither of these errors are things that the caller can really recover from. I would say these should be fatal panics as they indidcate a programmer bug.